server-manager.proto

```diff
@@ -237,33 +214,26 @@ message InspectStateResponse {
     optional bytes exception_data = 5; // Exception payload when finished with EXCEPTION
-    repeated Report reports = 6; // Reports produced while processing the query
+    repeated Output reports = 6; // Reports produced while processing the query
 }
 
 // Validity proof for an output
 message OutputValidityProof {
     uint64 input_index_within_epoch = 1; // Local input index within the context of the related epoch
     uint64 output_index_within_input = 2; // Output index within the context of the input that produced it
     CartesiMachine.Hash output_hashes_root_hash = 3; // Merkle root of all output hashes of the related input
-    CartesiMachine.Hash vouchers_epoch_root_hash = 4; // Merkle root of all voucher hashes of the related epoch
-    CartesiMachine.Hash notices_epoch_root_hash = 5; // Merkle root of all notice hashes of the related epoch
-    CartesiMachine.Hash machine_state_hash = 6; // Hash of the machine state claimed for the related epoch
-    repeated CartesiMachine.Hash output_hash_in_output_hashes_siblings = 7; // Proof that this output hash is in the output-hashes merkle tree. This array of siblings is bottom-up ordered (from the leaf to the root).
-    repeated CartesiMachine.Hash output_hashes_in_epoch_siblings = 8; // Proof that this output-hashes root hash is in epoch's output merkle tree. This array of siblings is bottom-up ordered (from the leaf to the root).
-}
-
-enum OutputEnum {
-    VOUCHER = 0;
-    NOTICE = 1;
+    CartesiMachine.Hash outputs_epoch_root_hash = 4; // Merkle root of all outputs_epoch_root_hash for inputs in epoch
+    CartesiMachine.Hash machine_state_hash = 5; // Hash of the machine state claimed for the related epoch
+    repeated CartesiMachine.Hash output_hash_in_output_hashes_siblings = 6; // Proof that this output hash is in the output-hashes merkle tree. This array of siblings is bottom-up ordered (from the leaf to the root).
+    repeated CartesiMachine.Hash output_hashes_in_epoch_siblings = 7; // Proof that this output-hashes root hash is in epoch's output merkle tree. This array of siblings is bottom-up ordered (from the leaf to the root).
 }
 
 // Data that can be used as proof to validate notices and execute vouchers on the base layer blockchain
 message Proof {
     uint64 input_index = 1; // Index of input starting from genesis
     uint64 output_index = 2; // Index of output (voucher or notice) in the context of the input
-    OutputEnum output_enum = 3; // Type of the output
-    OutputValidityProof validity = 4; // Validity proof for an output
-    bytes context = 5; // Data that allows the validity proof to be contextualized within submitted claims. Currently, the context is the epoch number as a ABI-encoded uint256.
+    OutputValidityProof validity = 3; // Validity proof for an output
+    bytes context = 4; // Data that allows the validity proof to be contextualized within submitted claims. Currently, the context is the epoch number as a ABI-encoded uint256.
```

Comment:
Just an observation, for `rollups-contracts` v2, we don't need this `context` field anymore. Now, in order to retrieve the correct claim from the consensus, you only need to provide the application contract address (like before), and the input range (first input index and last input index). So, maybe we can change this field here to be that? We can discuss this further in the Output Unification meeting too. :-)

Reply:
Hmm. The input range is easy. I am sure we have this information lying arround. But would the Server Manager be informed, during session startup, of this contract address? And should we bundle it in the proof structure itself?

Reply:
I think the node already has hold of the application address, since it needs to filter out `InputAdded` events for a specific application address.

Reply:
Yes, the node has the app address. It is reasonable for the node to append the app address to the Proof, so we don't need to pass it to the server manager.
On the other hand, It would be good if the server manager added the input range to the proof.

Reply:
This is already for the "partial" output unification?

Reply:
Yes

Reply:
Do you need these as uint256 or can I just give you uint64's?

Reply:
It depends on what the blockchain code needs. @guidanoli, what should the GraphQL output (a JSON) look like?

Reply:
@gligneul I think the `InputRange` struct can be on the `FinishEpochResponse` itself, since it would be the same for every output in the same epoch, right?

Reply:
How would the front end get the input range if we do that? There is no epoch structure in the GraphQL API.

Reply:
We already include stuff that would be the same for every output. I guess the idea was to give exactly what the user would need. Rather than have the user fish around for the required data. Has this understanding been changed?

Reply:
> Has this understanding been changed?
No, it has not. Let's discuss this in more detail in the workgroup meeting.

Reply:
Alright, adding this to our agenda. :notebook: